basic_module/account/src/import.rs

```diff
@@ -0,0 +1,19 @@
+use ckey::{Ed25519Public as Public, Signature};
+
+pub trait FeeManager {
+    fn accumulate_block_fee(&self, total_additional_fee: u64, total_min_fee: u64);
+}
+
+#[allow(dead_code)]
+pub fn fee_manager() -> Box<dyn FeeManager> {
+    unimplemented!()
+}
+
+pub trait SignatureManager {
```

Comment:
`SignatureManager` is not imported from other modules but provided by this module. BTW in the current FML, we do not need such a distinction: exported or imported.

Reply:
Oh.. This is a huge mistake. Thank you.